include/rocksdb/options.h

```diff
@@ -942,6 +942,9 @@ struct DBOptions {
   // relies on manual invocation of FlushWAL to write the WAL buffer to its
   // file.
   bool manual_wal_flush = false;
+
+  // If true flushes all column families atomically.
```

Comment:
Need to be clearer about what it means. Explain in what scenario it will be useful. I think we need to mention that to users it is only applicable if they don't write to WAL.

Reply:
Done. Will push later.